cpp/ql/test/library-tests/dataflow/external-models/validatemodels.expected

```diff
@@ -16,10 +16,13 @@
 | Dubious signature "(..(*)(..))" in summary model. |
 | Dubious signature "(..(*)(..),..(*)(..),..(*)(..),..(*)(..))" in summary model. |
 | Dubious signature "(..(*)(..),..(*)(..),..(*)(..),void *)" in summary model. |
+| Dubious signature "(..(*)(..),const void *,unsigned int)" in summary model. |
```

Comment:
Maybe we should actually start fixing this dubious signature thing, because this file is just getting bigger and bigger without being truly useful.